web/src/message_view_header.ts

```diff
@@ -175,9 +175,11 @@ export function render_title_area(): void {
 // This function checks if "modified_sub" which is the stream whose values
 // have been updated is the same as the stream which is currently
 // narrowed and rerenders if necessary
-export function maybe_rerender_title_area_for_stream(modified_sub: StreamSubscription): void {
+export function maybe_rerender_title_area_for_stream(modified_sub: StreamSubscription | number): void {
     const current_stream_id = narrow_state.stream_id();
-    if (current_stream_id === modified_sub.stream_id) {
+    const stream_id = typeof modified_sub === "number" ? modified_sub : modified_sub.stream_id
```

Comment:
I meant just accepting stream ID and not `StreamSubscription` object as eventually we use only ID.

Reply:
Aha, ok